### PR TITLE
add print-run

### PR DESCRIPTION
**Pull request by rlcee**
- remove "print-cid"
- add "print-run" which provides info about what is in a calibration set for a given run.
- options for print-run include drilling down to cid or table content
- all parse-able output is now available by string via getResult(), this enables imbedding DbTool in other code

### CODE REVIEW

**Comment by rlcee** [comment]
> Any suggestions for reviewers for this PR?

Changes will force minor updates to Richie's code

**Comment by kutschke** [comment]
@bonventre I added you as a reviewer because Ray says that this will require updates to your code.  Please review this.  Also, I will merge soon after you approve - so pick a time when you are ready to deal with the fallout.

**Review by bonventre — APPROVED** [review]

**Comment by rlcee** [comment]
> Hi Ray, looks like if I now do print-run with the content flag , getResult() will then give me the same thing as calling findCid and returnContent(cid) would? Should be easy to edit my side. Feel free to merge this whenever, the only dependence is TrackerCalibration which I can fix later.

Correct.

**Pull request merged**
